iphone/Maps/Bookmarks/BookmarksList/BookmarksListInfoViewController.swift

```diff
@@ -26,17 +26,23 @@ final class BookmarksListInfoViewController: UIViewController {
   
   override func viewDidLoad() {
     super.viewDidLoad()
-    descriptionButton.setTitle(L("placepage_place_description").uppercased(), for: .normal)
     separatorsConstraints.forEach { $0.constant = 1 / UIScreen.main.scale }
 
     guard let info = info else { return }
     updateInfo(info)
   }
 
-  private func updateInfo(_ info: IBookmakrsListInfoViewModel) {
+  private func updateInfo(_ info: IBookmarksListInfoViewModel) {
     titleLabel.text = info.title
     descriptionButton.isHidden = !info.hasDescription
-
+    descriptionButton.isEnabled = info.isHtmlDescription
+    if info.hasDescription {
+      var description = info.isHtmlDescription
+      ? BookmarksListInfoViewController.getPlainText(info.description)
+      : info.description
+      descriptionButton.setTitle(description?.uppercased(), for: .normal)
```

Comment:
```suggestion
      let description = info.isHtmlDescription
          ? BookmarksListInfoViewController.getPlainText(info.description)
          : info.description
      descriptionButton.setTitle(description for: .normal)
```
There is no need to uppercase it.